lib/src/widgets/slidable.dart

```diff
@@ -776,13 +786,22 @@ class SlidableState extends State<Slidable>
         }
       }
     });
+    if (widget.controller != null && widget.controller.onSliding != null) {
```

Comment:
To modify once the callbacks will be on the Slidable and not the SlidableController